tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/JCloudLedgerOffloaderFactory.java

```diff
@@ -24,6 +24,8 @@
 import org.apache.bookkeeper.common.util.OrderedScheduler;
 import org.apache.bookkeeper.mledger.LedgerOffloaderFactory;
 import org.apache.bookkeeper.mledger.offload.jcloud.impl.BlobStoreManagedLedgerOffloader;
+import org.apache.bookkeeper.mledger.offload.jcloud.provider.JCloudBlobStoreProvider;
+import org.apache.bookkeeper.mledger.offload.jcloud.provider.TieredStorageConfiguration;
 
 /**
  * A jcloud based offloader factory.
```

Comment:
This class seems to be dead code. It's not referenced from anywhere else.

Reply:
I agree, but it isn't a class I added to the code base as part of the refactor. It was there previously, so I just left it in. As far as I can tell, it was added in the 2.2.0 release.  It appears to be the code that is responsible for instantiating the tiered storage offloader